api/sign.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+var sjcl = require('ripple-lib').sjcl;
+var Seed = require('ripple-lib').Seed;
+var SerializedObject = require('ripple-lib').SerializedObject;
+var validate = require('./lib/validate');
+
+var HASH_TX_ID = 0x54584E00; // 'TXN'
```

Comment:
Can we prefer using this: https://github.com/ripple/ripple-lib/blob/447ae3f38ff121124af37f2beda6921a1936ad01/src/js/ripple/hashprefixes.js